<ui version="4.0" >
 <class>MoveCopyStudyDialog</class>
 <widget class="QDialog" name="MoveCopyStudyDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>777</width>
    <height>820</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" >
   <item row="0" column="0" >
    <layout class="QGridLayout" >
     <property name="leftMargin" >
      <number>0</number>
     </property>
     <property name="topMargin" >
      <number>0</number>
     </property>
     <property name="rightMargin" >
      <number>0</number>
     </property>
     <property name="bottomMargin" >
      <number>0</number>
     </property>
     <property name="horizontalSpacing" >
      <number>6</number>
     </property>
     <property name="verticalSpacing" >
      <number>6</number>
     </property>
     <item row="1" column="0" >
      <layout class="QGridLayout" >
       <property name="leftMargin" >
        <number>0</number>
       </property>
       <property name="topMargin" >
        <number>0</number>
       </property>
       <property name="rightMargin" >
        <number>0</number>
       </property>
       <property name="bottomMargin" >
        <number>0</number>
       </property>
       <property name="horizontalSpacing" >
        <number>6</number>
       </property>
       <property name="verticalSpacing" >
        <number>6</number>
       </property>
       <item row="0" column="0" >
        <layout class="QGridLayout" >
         <property name="leftMargin" >
          <number>0</number>
         </property>
         <property name="topMargin" >
          <number>0</number>
         </property>
         <property name="rightMargin" >
          <number>0</number>
         </property>
         <property name="bottomMargin" >
          <number>0</number>
         </property>
         <property name="horizontalSpacing" >
          <number>6</number>
         </property>
         <property name="verticalSpacing" >
          <number>6</number>
         </property>
         <item row="0" column="0" colspan="2" >
          <widget class="QLabel" name="textLabel3" >
           <property name="text" >
            <string>2) Click 'Refresh List' and select from the available studies on &lt;br>your source folder</string>
           </property>
           <property name="alignment" >
            <set>Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="3" column="0" >
          <spacer>
           <property name="orientation" >
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeType" >
            <enum>QSizePolicy::Expanding</enum>
           </property>
           <property name="sizeHint" >
            <size>
             <width>20</width>
             <height>331</height>
            </size>
           </property>
          </spacer>
         </item>
         <item rowspan="3" row="1" column="1" >
          <widget class="Q3ListBox" name="studiesListBox" >
           <property name="selectionMode" >
            <enum>Q3ListBox::Multi</enum>
           </property>
          </widget>
         </item>
         <item row="1" column="0" >
          <widget class="QPushButton" name="refreshButton" >
           <property name="text" >
            <string>&amp;Refresh List</string>
           </property>
           <property name="shortcut" >
            <string>Alt+R</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0" >
          <widget class="QPushButton" name="selectAllButton" >
           <property name="text" >
            <string>&amp;Select All</string>
           </property>
           <property name="shortcut" >
            <string>Alt+S</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="0" >
        <layout class="QGridLayout" >
         <property name="leftMargin" >
          <number>0</number>
         </property>
         <property name="topMargin" >
          <number>0</number>
         </property>
         <property name="rightMargin" >
          <number>0</number>
         </property>
         <property name="bottomMargin" >
          <number>0</number>
         </property>
         <property name="horizontalSpacing" >
          <number>6</number>
         </property>
         <property name="verticalSpacing" >
          <number>6</number>
         </property>
         <item row="1" column="0" >
          <layout class="QHBoxLayout" >
           <property name="spacing" >
            <number>6</number>
           </property>
           <property name="leftMargin" >
            <number>0</number>
           </property>
           <property name="topMargin" >
            <number>0</number>
           </property>
           <property name="rightMargin" >
            <number>0</number>
           </property>
           <property name="bottomMargin" >
            <number>0</number>
           </property>
           <item>
            <widget class="QLabel" name="textLabel6" >
             <property name="text" >
              <string>Destination:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="destinationLineEdit" />
           </item>
           <item>
            <widget class="QPushButton" name="browseDestinationButton" >
             <property name="text" >
              <string>&amp;Browse...</string>
             </property>
             <property name="shortcut" >
              <string>Alt+B</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="0" column="0" >
          <widget class="QLabel" name="destinationLabel" >
           <property name="text" >
            <string>3) Select your destination location and Click '%1'</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item row="0" column="0" >
      <layout class="QVBoxLayout" >
       <property name="spacing" >
        <number>6</number>
       </property>
       <property name="leftMargin" >
        <number>0</number>
       </property>
       <property name="topMargin" >
        <number>0</number>
       </property>
       <property name="rightMargin" >
        <number>0</number>
       </property>
       <property name="bottomMargin" >
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="textLabel1" >
         <property name="text" >
          <string>1) Select where you would like to copy the studies from</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" >
         <property name="spacing" >
          <number>6</number>
         </property>
         <property name="leftMargin" >
          <number>0</number>
         </property>
         <property name="topMargin" >
          <number>0</number>
         </property>
         <property name="rightMargin" >
          <number>0</number>
         </property>
         <property name="bottomMargin" >
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="textLabel2" >
           <property name="text" >
            <string>Source:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="sourceLineEdit" />
         </item>
         <item>
          <widget class="QPushButton" name="browseSourceButton" >
           <property name="text" >
            <string>&amp;Browse...</string>
           </property>
           <property name="shortcut" >
            <string>Alt+B</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="1" column="0" >
    <layout class="QHBoxLayout" >
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="backButton" >
       <property name="text" >
        <string>&lt; Back</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="finishButton" >
       <property name="text" >
        <string>Finish</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton" >
       <property name="text" >
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Q3ListBox</class>
   <extends>Q3Frame</extends>
   <header>q3listbox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>MoveCopyStudyDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>741</x>
     <y>795</y>
    </hint>
    <hint type="destinationlabel" >
     <x>363</x>
     <y>781</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>finishButton</sender>
   <signal>clicked()</signal>
   <receiver>MoveCopyStudyDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>644</x>
     <y>795</y>
    </hint>
    <hint type="destinationlabel" >
     <x>203</x>
     <y>788</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
